db/schema.rb

```diff
@@ -328,7 +330,7 @@
   create_table "statement_items", force: :cascade do |t|
     t.bigint "statement_id", null: false
     t.bigint "declaration_id", null: false
-    t.string "state", null: false
+    t.enum "state", null: false, enum_type: "declaration_status_enum"
```

Comment:
`state` here vs `status` above?

I'd probably lean towards `state` and drop the `_enum`.

Reply:
👍 for `state` in both, and removing `_enum`, however could we also keep the declaration migrations in the declaration pr if possible?